src/components/WalletModal/Option.tsx

```diff
@@ -14,21 +16,22 @@ const OptionCardLeft = styled.div`
   align-items: center;
 `
 
-const OptionCardClickable = styled.button<{ isActive?: boolean; clickable?: boolean }>`
+const OptionCardClickable = styled.button<{ clickable: boolean; selected: boolean }>`
   background-color: ${({ theme }) => theme.backgroundModule};
+  border: none;
   width: 100% !important;
-  border-color: ${({ theme, isActive }) => (isActive ? theme.accentActive : 'transparent')};
 
   display: flex;
   flex-direction: row;
   align-items: center;
   justify-content: space-between;
-  margin-top: 2rem;
-  padding: 1rem;
+  padding: 18px;
```

Comment:
`isActive` was unused/leftover from old code, so I removed the border completely since there is no border in our current design. I upped the padding from 16px (idk why we had rem here) to 18px to fix the size change caused by zero border

Reply:
Sounds good, thanks for explanation!